src/Microsoft.ML.Data/Transforms/TransformsExtensionsCatalog.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML
+{
+    using HashDefaults = Transforms.HashEstimator.Defaults;
+
+    /// <summary>
+    /// Extensions for Column Copying Estiamtor.
+    /// </summary>
+    public static class ColumnCopyingCatalog
+    {
+        /// <summary>
+        /// Copies the input column to another column named as specified in <paramref name="outputColumn"/>.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="inputColumn">The source column.</param>
```

Comment:
><param name="inputColumn">The source column.</param> [](start = 12, length = 52)

can we have same description as in hashing estimator:
<param name="inputColumn">Name of the input column.</param>